app/Views/admin/certificates/CertificateCreate.php

```diff
@@ -23,13 +23,17 @@
                         <i class="ri-arrow-left-fill label-icon align-middle rounded-pill fs-16 me-2"></i>Volver
                     </a>
                 </div>
-                <form id="createproduct-form" autocomplete="off" class="needs-validation" method="POST">
+                <form id="form" autocomplete="off" class="needs-validation" novalidate method="POST">
```

Comment:
Why you added novalidate?
```suggestion
                <form id="form" autocomplete="off" class="needs-validation" method="POST">
```